grails-app/views/public/show.gsp

```diff
@@ -11,7 +11,8 @@
 
         <script type="text/javascript" language="javascript" src="https://www.google.com/jsapi"></script>
 
-        <asset:javascript src="public-show.js" />
+        <asset:javascript src="public-show" />
+        <asset:stylesheet src="public-show" />
```

Comment:
Are you sure you don't need file extensions?

Reply:
I'm sure.